### DISCUSSION TITLE
Add description to Secrets and Variables

### DISCUSSION THREAD

**Discussion by andreaangiolillo** [community/community#57527] [Code Security]
### Select Topic Area

Product Feedback

### Body

Hello 👋 

It would be nice to allow users to include brief descriptions for the Secrets or Variables utilized in a GitHub Action. This would provide context and clarify the purpose of each Secret or Variable.

Thanks

**Comment by renrutsirhc** [comment]
This would be a really useful feature!

**Comment by saasaTom** [comment]
This would save us having to document what secrets are elsewhere.  Not the value, but simply where to find the correct value or some information around what the value should be. i.e. A link to a key vault or password manager.
